### ISSUE TITLE
ffmpeg never end : overwrite 

### ISSUE THREAD

**Issue by wadouk** [sitespeedio/browsertime#464]
I have those lines in log
```
[2018-02-26 14:36:52] DEBUG: [root] Stopped ffmpeg
[2018-02-26 14:36:52] DEBUG: [root] Converting video to viewable format.
```
And it never ends

I succeed to find the process in container and kill the process
So I have the log that continue with
```
[2018-02-26 14:38:23] DEBUG: [root] Telling browser to quit.
[2018-02-26 14:38:23] ERROR: [sitespeedio.queuehandler] Error: Command failed: ffmpeg -i /sitespeed.io/dist/pages/.../data/video/0.mp4 -c:v libx264 -crf 23 -preset medium -vf format=yuv420p /sitespeed.io/dist/pages/.../data/video/tmp.mp4
```

When I retry the command in another shell, it prompt me to overwrite the file

A workaround I experiment is to prune the dist directory before running all.

Version : sitespeed 6.2.3 but also 6.3.5
the site is in a private network, so usefulness to give it

**Comment by soulgalore** [comment]
Hi @wadouk thanks for reporting, can you share how you run, base OS (where you run the container etc) so we can try to reproduce? 
Thanks
Peter

**Comment by wadouk** [comment]
Hi,
Linux 3.16.0_4 debian 8.2
On the same host, i have 3 builds, dev, staging, production, only the dev one had this problem. Seems the tmp file doesn't remove after processing.
Runned with a constrained network, but doesn't seem to be relevant for this problem : without it was the same.
It could be great to dump output streams of child processes to investigate easier.

**Comment by soulgalore** [comment]
Hi @wadouk the ffmpeg handling was changed in 3.0, can you test when you have time and see if it works better now?

Best
Peter

**Comment by soulgalore** [comment]
We released 4.0 and this has changed a lot. Please re-open if this is still a issue.
